web/profiles/joinup/config/install/block.block.navigation_bottom.yml

```diff
@@ -1,37 +1,34 @@
 langcode: en
 status: true
 dependencies:
+  config:
+    - og_menu.ogmenu.navigation
   module:
-    - custom_page
-    - node
+    - collection
     - og
   theme:
     - joinup_theme
-id: menusubpages
+id: navigation_bottom
```

Comment:
Blocks can be placed anywhere on the page so putting a position in the machine name is a bit pointless. Let's call this the `table_of_contents`.